src/popup/App.vue

```diff
@@ -130,24 +130,39 @@ export default {
 </script>
 
 <style lang="scss">
-html {
-  height: 100%;
+html,
+body {
+  width: 360px;
```

Comment:
Does it make sense to extract the fixed width into a variable? There are more than a few occurrences, so far.